documentation/using_plugins.md

```diff
@@ -126,7 +126,14 @@ json.dump({'value': secret}, sys.stdout)
 
 ## PuppetDB
 
-The PuppetDB plugin supports looking up targets from PuppetDB. It takes a `query` field, which is either a string containing a [PQL](https://puppet.com/docs/puppetdb/latest/api/query/v4/pql.html) query or an array containing a [PuppetDB AST](https://puppet.com/docs/puppetdb/latest/api/query/v4/ast.html) format query. The query is used to determine which targets should be included in the group. If `name` or `uri` is not specified with a [fact lookup](https://github.com/adreyer/bolt/blob/c4891ad27a8fd871f04df1e17ef5e295a3cab7f4/pre-docs/using_plugins.md#fact-lookups) then the `[certname]` for each target in the query result will be used as the `uri` for the new target. Read the [Migrating to Version 2](https://github.com/adreyer/bolt/blob/c4891ad27a8fd871f04df1e17ef5e295a3cab7f4/pre-docs/using_plugins.md#migrating-to-version-2) section for more details on `uri` and `name` keys.
+The PuppetDB plugin supports looking up targets from PuppetDB. It takes a `query` field, which is
+either a string containing a [PQL](https://puppet.com/docs/puppetdb/latest/api/query/v4/pql.html)
+query or an array containing a [PuppetDB
+AST](https://puppet.com/docs/puppetdb/latest/api/query/v4/ast.html) format query. The query is used
+to determine which targets should be included in the group. If `name` or `uri` is not specified with
+a fact lookup then the `[certname]` for each target in the query result will be used as the `uri`
```

Comment:
Seems like removing the link to `fact lookup` is correct as there is no header for that. Do we need to add a section for that?

Reply:
@adreyer @K8med ^^  I don't think so, 'fact lookup' feels like it's relatively self explanatory in context? But that could also be because I know what we mean :stuck_out_tongue:

Reply:
Since we don't describe it until lower I think something more could be said at this point.

Reply:
Ok, added another header level and linked this to that.